data_steward/tools/create_ehr_and_rdr_snapshot.sh

```diff
@@ -9,6 +9,7 @@ Usage: create_ehr_snapshot.sh
   --rdr_dataset <RDR dataset ID>
   --validation_dataset <Validation dataset ID>
   --dataset_release_tag <release tag for the CDR>
+  --snapshot_date <date snapshot is generated format yyyy-mm-dd>
```

Comment:
I don't think this was specified in the ticket. A general question though, should we provide the `snapshot_date` as a parameter or should we get the date using `snapshot_date=$(date +'%Y-%m-%d')` in the bash script directly?

Reply:
Oh, to be honest, I didn't know there was another way to do it 😅 I think it would be better if we didn't need to supply it as a parameter.